alf/environments/suite_gym.py

```diff
@@ -112,4 +124,8 @@ def wrap_env(gym_env,
     for wrapper in torch_env_wrappers:
         env = wrapper(env)
 
+    # # To apply channel_first transpose on torch env
+    # if image_channel_first:
+    #     env = torch_wrappers.ImageChannelFirst(env)
+
```

Comment:
remove?

Reply:
I left it there for information of how to use it as an alternative of channel_first transpose, but I will remove it and add the something into the comments.